INSTALL.md

```diff
@@ -91,12 +91,12 @@ Then you can run `make` to build the source.
 
     $ make
 
-It builds uftrace and resulting binary exists in the current directory.
+It builds uftrace, placing the resulting binary in the current directory.
```

Comment:
`make` produces `uftrace` and `libmcount*.so` shared objects so better to be plural here.
```suggestion
It builds uftrace, placing the resulting binaries in the current directory.
```